azerothcore/azerothcore-wotlk
# style(Core/Achievement): Improve error msg for achievement_reward

## CHANGES PROPOSED:
- Change a outError to outDebug
- Fix typos and improve the texts to waste less time for the user
- Added some slightly useful comments for future devs  


## ISSUES ADDRESSED:
- Closes 



## TESTS PERFORMED:
- Can't test at the moment, but it's mostly "cosmetic" changes




## HOW TO TEST THE CHANGES:

- Not much to test really, better review the code directly
- Insert various rows in the achievement_reward table with missing values (cf http://www.azerothcore.org/wiki/achievement_reward note: I updated the wiki too) and see the console messages when typing `.reload achievement_reward`. For the outDebug, you'll need to enable debug in worldserver.conf


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
